### ISSUE TITLE
visibility scale bug?

### ISSUE THREAD

**Issue by jtatusko** [deepmind/pysc2#198]
```python
class Visibility(enum.IntEnum):
  """Values for the `visibility` feature layers."""
  HIDDEN = 0
  SEEN = 1
  VISIBLE = 2
```
https://github.com/deepmind/pysc2/blob/master/pysc2/lib/features.py#L262

should scale be set to three, since there are only three possible values of visibility, or am i misunderstanding the purpose of the scale property?

**Comment by jtatusko** [comment]
Same for effects enum. The effects are given scale 16 but there are only 12 possible values in the enum

**Comment by jtatusko** [comment]
Should the effects enum also start at 0 like the rest?

**Comment by tewalds** [comment]
You are correct. Unfortunately this is tough for us to fix given that our agents depend on this... Sorry.
